source/documentation/make-updates-during-tax-year.html.md

```diff
@@ -166,11 +174,19 @@ If a customer wants to make a change to a previously submitted periodic update,
 
 When a quarterly update has been made, the customer can view a summary of the updates made to date. To retrieve this summary, the software should call any of the following endpoints, depending on the business income type:
 
+<<<<<<< HEAD
+- [Retrieve a Self-Employment Cumulative Period Summary](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/self-employment-business-api/4.0/oas/page#tag/Self-Employment-Cumulative-Period-Summaries/paths/~1individuals~1business~1self-employment~1%7Bnino%7D~1%7BbusinessId%7D~1period~1%7BtaxYear%7D~1%7BperiodId%7D/get)
```

Comment:
Please fix link to https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/self-employment-business-api/4.0/oas/page#tag/Self-Employment-Cumulative-Period-Summary/paths/~1individuals~1business~1self-employment~1%7Bnino%7D~1%7BbusinessId%7D~1cumulative~1%7BtaxYear%7D/get, thanks